use BPF_MAP_TYPE_LPM_TRIE for range matching (#11526)
PowerDNS/pdns: pdns/dnsdistdist/docs/reference/ebpf.rst

    :param ComboAddress address: The address to block

  .. method:: BPFFilter:blockRange(Netmask)

  .. versionchanged:: 1.8.0

>     Block ip in this range. 

Reviewer: ```suggestion
    Block all IP addresses in this range. 
```

>     DNSDist check block range after check block ip before check block DNSName. 

Reviewer: ```suggestion
    DNSDist eBPF code first checks if an exact IP match is found, then if a range matches, and finally if a DNSName does.
```

    :param ComboAddress address: The address to block

  .. method:: BPFFilter:addRangeRule(Netmask , action [, force=false])

>   .. versionchanged:: 1.8.0

Reviewer: ```suggestion
    .. versionadded:: 1.8.0
```

    :param ComboAddress address: The address to block

  .. method:: BPFFilter:addRangeRule(Netmask , action [, force=false])

  .. versionchanged:: 1.8.0

    Block all IP addresses in this range. 

    DNSDist eBPF code first checks if an exact IP match is found, then if a range matches, and finally if a DNSName does.

    :param string Netmask: The ip range to block or unblock

>     :param int action: set ``action``  to ``0`` to unblock a range, set ``action`` to ``1`` to block a range.

Reviewer: I wonder if something like "set ``action`` to ``0`` to allow a range, set ``action`` to ``1`` to block a range." would not be clearer?
Shouldn't we document the "2"/"truncate" option there?

Author: Oh, yeah, I guess that makes sense.

    :param ComboAddress address: The address to unblock

  .. method:: BPFFilter:unblockRange(Netmask)

  .. versionchanged:: 1.8.0

>     Unblock ip in this range. 

Reviewer: ```suggestion
    Unblock this range. 
```

>     DNSDist check unblock range after check block ip before check block DNSName. 
>     If ip is unblocked by this method but blocked by DNSName, the packet will be accept.
>     If ip is unblocked by this method but blocked by ``block`` , the packet will be blocked.

Reviewer: I'm afraid I don't really understand the last sentence?
```suggestion
    DNSDist eBPF code first checks if an exact IP match is found, then if a range matches, and finally if a DNSName does.
    If a query coming from an IP or range marked as allowed, but is for a qname for which a DNSName block exists, the packet will be accepted as the first match wins.
```

    :param ComboAddress address: The address to unblock

  .. method:: BPFFilter:rmRangeRule(Netmask)

  .. versionchanged:: 1.8.0

    :param Netmask string: The rule you want to remove

  .. method:: BPFFilter:lsRangeRule()

>   .. versionchanged:: 1.8.0

Reviewer: ```suggestion
    .. versionadded:: 1.8.0
```
